src/components/annotations/draw.js

```diff
@@ -168,7 +168,6 @@ function drawOne(gd, index) {
     var font = options.font;
 
     var annText = annTextGroupInner.append('text')
-        .classed('annotation', true)
```

Comment:
I think you've hit on a flaw in how github displays comments... I'm reading:

> This commit is reverted in ababc49 to make the tests added in https://github.com/plotly/plotly.js/pull/1683 :white_check_mark: 

but I'm looking at https://github.com/plotly/plotly.js/pull/1638/commits/a31549d3b7532c48200289607ab435e4e80d891f which came after ababc49 and rereverts the change originally made in https://github.com/plotly/plotly.js/commit/1a8f1ed1e855fb9157f728c191b00784fe3f1258 💫 

Anyway, totally approve of this change. My best guess is these got the same `.annotation` class because at some point in the distant past they weren't all encapsulated in one group like they are now, and I wanted to be able to more easily select the whole annotation. Has always been confusing to deal with...